app/views/internal/configs/show.html.erb

```diff
@@ -679,6 +679,26 @@
           </div>
         </div>
 
+        <div class="card mt-3">
+          <%= render partial: "card_header",
+                     locals: {
+                       header: "Sponsors",
+                       state: "collapse",
+                       target: "sponsorsContainer",
+                       expanded: "false"
+                     } %>
+          <div id="sponsorsContainer" class="card-body collapse hide" aria-labelledby="sponsorsContainer">
+            <div class="form-group">
+              <%= f.label :sponsor_headline %>
+              <%= f.text_field :sponsor_headline,
+                               class: "form-control",
+                               value: SiteConfig.sponsor_headline,
+                               placeholder: "Community Sponsors" %>
```

Comment:
The default headline is Community Sponsors but is configurable. should we not do the same for the placeholder?